### DISCUSSION TITLE
Toast

### DISCUSSION THREAD

**Discussion by asolux** [tabler/tabler#848] [Q&A]
Somehow I can't handle the _toasts_. Can someone please give me a start?
If I set the css class `show`, the automatic hide does not work.
How can I show the message via javascript?
Thanks!  👋

**Comment by schophel** [comment]
I used this content to solve a similar issue:
[https://stackoverflow.com/questions/63515279/how-to-initialize-toasts-with-javascript-in-bootstrap-5](https://stackoverflow.com/questions/63515279/how-to-initialize-toasts-with-javascript-in-bootstrap-5)

**Comment by ThatoFrancis** [comment]
Hello, 
how can I change a Toast's color in a .cs file without adding a class property to its component?
Thank you.

**Reply by ethancrawford** [reply]
@ThatoFrancis - inspecting the source code on the toasts page shows that toasts get their colours from the `--tblr-toast-color` and `--tblr-toast-bg` css variables (which appear to be defined in Bootstrap's _toasts.scss file).

**Comment by ethancrawford** [comment]
@asolux - are you still wondering about autohide for a toast?
I believe you are correct in observing that to use autohide, you need to start with the toast not visible, and trigger it later. As for how to do that with javascript, have you read through the Bootstrap documentation? https://getbootstrap.com/docs/5.3/components/toasts for example may help. If you have, and you're still having issues, perhaps you could describe a little more clearly the exact issue you're having.
